pmdarima/arima/tests/test_arima.py

```diff
@@ -768,7 +768,7 @@ def test_warn_for_large_differences():
 def test_warn_for_stepwise_and_parallel():
     with pytest.warns(UserWarning):
         auto_arima(lynx, suppress_warnings=False, d=1,  # noqa: F841
-                       error_action='ignore', stepwise=True, n_jobs=2)
+                   error_action='ignore', stepwise=True, n_jobs=2)
```

Comment:
Not sure why the linter complained about this... I did not edit it until the linter complained.

Reply:
Flake8 had a release a [few days ago](https://flake8.pycqa.org/en/latest/release-notes/index.html), so maybe something with that

Reply:
Ha, that's funny. I found the same thing in another codebase I manage. `flake8` now complains if you have an F-string with no formatting fields in it. Good catch, I'm sure that probably fit the other fella who submitted a PR